docs/SQL/extensibility/dependencies.md

```diff
@@ -0,0 +1,107 @@
+
+:::info Invalid coordinates
+
+In rare cases, your dependency might be only accessible to the cluster or the build system but not Prophecy itself. If you're confident that the dependency is correct, but the validation error shows up, it's safe to press **Save Anyways** to ignore that warning.
+
+:::
+
+### Add a DBT dependency
+
+To add a DBT dependency, you must select **DBT Hub** and provide the dbt package and version number.
+
+![Add a DBT Dependency](img/dependencies-dbt.png)
+
+You can find the dbt package and version number from the [dbt Package hub](https://hub.getdbt.com/dbt-labs/dbt_utils/latest/).
+
+![dbt_utils dependency](img/dependencies-dbt-utils.png)
+
+You can copy and paste the information into the Create Dependency fields.
+
+### Add a GitHub dependency
+
+To add a GitHub dependency, you must select **GitHub** and provide the Git Repository, Revision, and Sub Directory.
+
+![Add a GitHub Dependency](img/dependencies-github.png)
+
+The Revision must be either a Git tag, commit has, or branch name.
+
+### Add a Prophecy Project dependency
+
+To add another Prophecy Project as a dependency, you must select **Prophecy Project** and select a project from the Project Depedencies dropdown.
+
+:::note
+
+The Prophecy Project must already be released for it to show up in the dropdown.
+
+:::
+
+![Add a Prophecy Project Dependency](img/dependencies-prophecy-project.png)
+
+## Use a dependency
+
+Once you've added a dependency, you can use the following entities from them:
+
+- Models
+- Seeds
+- Functions
+- Gems
+- Data Tests
+
+![Use a dependency](img/dependencies-use.png)
+
+Simply drag and drop the entity that you'd like to use from the dependencies section onto your Visual canvas.
+
+## Delete dependency
+
+To delete a dependency, first navigate to the dependencies list on the [Manage Dependencies](#view-dependencies) screen. There you can click on a trash icon next to the dependency that you'd like to delete.
+
+:::caution Deleting a dependency
+
+Deleting a dependency deletes it within the whole Project, and all inheriting Models. If you're not confident about
+the functionality of the other Models, it's usually better to disable a dependency, instead of deleting it.
+
+:::
+
+## Installation on a cluster
+
+Whenever you connect the Models to a cluster, dependencies are automatically installed on your cluster. If the cluster doesn't have the dependency installed yet, Prophecy installs it and restarts the cluster automatically.
+
+## Storage
+
+All of your dependencies are stored at the Project-level. When adding a dependency to a single Model, by default it becomes available to all the other Models within the same Project.
+
+Dependencies are natively saved within your build-system files. E.g. if you're using Scala, they're saved in the **pom.xml** file, if you're using Python, they're saved in the **setup.py** file.
```

Comment:
This is not true for SQL.
For SQL we can say, it is stored in `packages.yml` file. `Dependencies are saved within your packages.yml file`